.storybook/preview.ts

```diff
@@ -0,0 +1,37 @@
+import { Preview } from '@storybook/react';
+import { IntlProvider } from 'react-intl';
+
+import features from '../examples/src/features';
+
+import '../src/styles/variables';
+import '../src/styles/base.scss';
+
+// Constants
+global.FEATURE_FLIPS = global.FEATURE_FLIPS || features; // cannot use the word FEATURES, storybook uses it
```

Comment:
nit: the comment could probably be removed. once the change is merged, the context isn't really needed anymore for future readers